src/sentry/rules/processing/processor.py

```diff
@@ -272,9 +299,13 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
 
         state = self.get_state()
         condition_list, filter_list = split_conditions_and_filters(rule.data.get("conditions", ()))
-
-        # Sort `condition_list` so that most expensive conditions run last.
-        condition_list.sort(key=lambda condition: is_condition_slow(condition))
+        slow_conditions = None
+        if features.has("organizations:process-slow-alerts", self.project.organization):
```

Comment:
Should we be processing the slow conditions if they have this flag?
Or should we be skipping it?

it looks like we're only processing the fast conditions here?

Reply:
If they don't have the flag they just process them in order of fast to slow like it's done now

Reply:
Yea - the flag makes it sound like we should _only_ be processing them IF they have the flag.

kinda wish it was named better but 🤷